app/util/static.go

```diff
@@ -0,0 +1,84 @@
+package util
+
+import (
+	"sync"
+	"github.com/raggaer/castro/app/database"
+	"path/filepath"
+	"os"
+	"strings"
+)
+
+var (
+	// ExtensionStatic holds all extension subtopic static folders
+	ExtensionStatic = &StaticList{
+		list: []string{},
+	}
+)
+
+// StaticList struct used to hold static lists
+type StaticList struct {
+	rw sync.RWMutex
+	list []string
+}
+
+// FileExists checks if the given resource exists
+func (e *StaticList) FileExists(url string) bool {
+	// Read lock mutex
+	e.rw.RLock()
+	defer e.rw.RUnlock()
+
+	// Loop list
+	for _, e := range e.list {
+
+		if strings.HasPrefix(strings.Replace(url, "/", "\\", -1), filepath.Join(e)) {
```

Comment:
This only seems to work on Windows. Linux (and OSX?) uses forwardslashes in file paths. Possible fix:
`if strings.HasPrefix(url, filepath.Join(e)) || strings.HasPrefix(strings.Replace(url, "/", "\\", -1), filepath.Join(e)) {`